E2_ESRGAN/lib/phase_1.py

```diff
@@ -0,0 +1,68 @@
+from functools import partial
+import tensorflow as tf
+from lib import settings, utils, model, dataset
+
+class Model:
+  def __init__(self, **kwargs):
+  """ PSNR model training on L1 Loss to warmup the Generator
+      Args:
+        data_dir: Path to store the downloaded dataset from tfds.
+        summary_writer: SummaryWriter object to write summaries for Tensorboard
+  """
+    settings = settings.settings()
+    self.warmup_num_iter = settings.get("warmup_num_iter", None)
+    dataset_args = settings["dataset"]
+    self.phase_args = settings["train_psnr"]
+    self.dataset = dataset.load_dataset(
+        dataset_args["name"],
+        dataset.scale_down(
+            method=dataset_args["scale_method"],
+            dimension=dataset_args["dimension"]),
+        batch_size=settings["batch_size"],
+        data_dir=kwargs["data_dir"])
+    self.G = model.RRDBNet(out_channel=3)
+    self.iterations = settings["iterations"]
```

Comment:
Try to have `_iterations` kind of naming for attributes that don't need to be public.

Reply:
Done :)